CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ Latency is another important factor for Alacritty. On X11, Windows, and macOS th
 Code should be documented where appropriate. The existing code can be used as a guidance here and
 the general `rustfmt` rules can be followed for formatting.
 
-If any change has been made to the `config.rs` file, it should also be documented in the man pages.
+If any change has been made to the `config.rs`, `ui_config.rs` files, it should also be documented in the man pages.
```

Comment:
I'm not sure that this should be changed that way.  And I don't think it's a part of this patch in the first place.

Reply:
Reverted.